src/request/remove-key/index.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html>
+
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <meta http-equiv="Content-Security-Policy"
-          content="default-src 'self' 'unsafe-eval' https://cdn.nimiq-testnet.com;
-          img-src http: https: blob: data:;
-          child-src 'self' blob:;
-          style-src 'self' https://fonts.googleapis.com;
-          font-src https://fonts.gstatic.com;">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-eval' https://cdn.nimiq-testnet.com;
```

Comment:
You changed it here, but did not change it in `src/derive-address/index.html`. Why these changes?

Reply:
No idea honestly. Will change it back